src/main/java/edu/kis/vh/nursery/defaultCountingOutRhymer.java

```diff
@@ -1,33 +1,66 @@
 package edu.kis.vh.nursery;
 
+/**
+ * Stack Class
+ */
 public class DefaultCountingOutRhymer {
 
-    private int[] NUMBERS = new int[12];
+    private final int STACK_MAX_SIZE = 12;
+    private final int INITIAL_STACK_INDEX=-1;
+    private int[] NUMBERS = new int[STACK_MAX_SIZE];
+    private int total = INITIAL_STACK_INDEX;
 
-    public int total = -1;
+    /**
+     *
+     * @return number of elements currently on the stack
+     */
+    public int getTotal() {
+        return total;
+    }
 
+    /**
+     * Puts element on  the stack
+     * @param in element to put on the stack
+     */
     public void countIn(int in) {
         if (!isFull())
             NUMBERS[++total] = in;
     }
 
-    public boolean callCheck() {
-        return total == -1;
+    /**
+     *
+     * Checks if stack is empty
+     * @return true if empty
+     */
+    boolean callCheck() {
+        return total == INITIAL_STACK_INDEX;
     }
 
-    public boolean isFull() {
-        return total == 11;
+    /**
+     *Checks if stack is full
+     * @return true if stack is full
+     */
+    boolean isFull() {
+        return total == STACK_MAX_SIZE-1;
     }
 
-    protected int peekaboo() {
+    /**
+     * returns  element from top of the stack
+     * @return element from top of the stack
+     */
+    int peekaboo() {
         if (callCheck())
-            return -1;
+            return INITIAL_STACK_INDEX;

```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)